include/internal/libspdm_macro_check.h

```diff
@@ -30,6 +39,10 @@
     #error If KEY_EX_CAP is enabled then at least one DHE algorithm must also be enabled.
 #endif
 
+#if !LIBSPDM_HASH_ALGO_SUPPORT
```

Comment:
It is valid use case that if SPDM only supports unsigned measurement with raw bit stream.